marklogic-data-hub/src/test/java/com/marklogic/hub/flow/impl/RunFlowWithStemmingEnabledTest.java

```diff
@@ -0,0 +1,61 @@
+package com.marklogic.hub.flow.impl;
+
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.DatabaseKind;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.flow.FlowInputs;
+import com.marklogic.hub.flow.RunFlowResponse;
+import com.marklogic.hub.impl.Versions;
+import com.marklogic.hub.job.JobStatus;
+import com.marklogic.mgmt.api.API;
+import com.marklogic.mgmt.api.database.Database;
+import com.marklogic.mgmt.resource.databases.DatabaseManager;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+public class RunFlowWithStemmingEnabledTest extends AbstractHubCoreTest {
+
+    Database db;
+
+    @BeforeEach
+    public void setupEach() {
+        HubConfig hubConfig = runAsDataHubDeveloper();
```

Comment:
I believe there's an issue here in that if the version of ML isn't supported, then the call to set stemmed searches to "off" won't be executed. So I'd just move this to the test. 

In fact, I think private methods can make this read nicely:

- enableAdvancedStemming();
- run the flow and verify an error occurs (I think it'd be good to assert on the contents of stepOutput too, as we can verify there are words in there indicating that a step definition was not found)
- reindexDatabase()
- run the flow again and verify it succeeds

Reply:
I think moving assumeTrue(new Versions(getHubConfig()).getMLVersion().getMajor() >= 10);  to the first line of setupEach () should resolve that issue right ?

Reply:
Since you only have the one test method, I think it's simpler to just put everything in that method. But if you want to keep the BeforeEach method, then I think assumeTrue will do the trick there.